src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -119,52 +195,115 @@ interface IOptionSettlementEngine {
+
+    /**
+     * @notice The protocol fee, expressed in basis points.
+     * @return The fee in basis points.
+     */
     function feeBps() external view returns (uint8);
 
-    // @return The address fees accrue to
+    /**
+     * @notice Returns the address to which protocol fees are swept.
+     * @return The address to which fees are swept
+     */
     function feeTo() external view returns (address);
 
-    // @return The balance of unswept fees for a given address
-    function feeBalance(address token) external view returns (uint256);
-
-    // @return The enum (uint8) Type of the tokenId
+    /**
+     * @notice Returns the token type (e.g. Option/Claim) for a given token Id
+     * @param tokenId The id of the option or claim.
+     * @return The enum (uint8) Type of the tokenId
+     */
     function tokenType(uint256 tokenId) external view returns (Type);
 
-    // @return The optionInfo Option struct for tokenId
+    /**
+     * @notice Returns Option struct details about a given tokenID if that token is
+     * a vToken.
+     * @param tokenId The id of the option.
+     * @return optionInfo The Option struct for tokenId.
+     */
     function option(uint256 tokenId) external view returns (Option memory optionInfo);
 
-    // @return The claimInfo Claim struct for claimId
+    /**
+     * @notice Returns Claim struct details about a given tokenId if that token is a
+     * claim NFT.
+     * @param tokenId The id of the claim.
+     * @return claimInfo The Claim struct for tokenId.
+     */
     function claim(uint256 tokenId) external view returns (Claim memory claimInfo);
 
-    // @notice Updates the address fees can be swept to
+    /**
+     * @notice Updates the address fees can be swept to.
+     * @param newFeeTo The new address to which fees will be swept.
+     */
     function setFeeTo(address newFeeTo) external;
 
-    // @return The tokenId if it exists, else 0
+    /**
+     * @notice Retrieves the optionId for the keccak256 hash of an Option struct
+     * with settlementSeed set to 0 at time of hashing.
+     * @param hash The keccak256 hash of an Option struct.
+     * @return optionId The tokenId if it exists, else 0.
+     */
     function hashToOptionToken(bytes32 hash) external view returns (uint256 optionId);
 
-    // @notice Sweeps fees to the feeTo address if there are more than 0 wei for each address in tokens
+    /**
+     * @notice Sweeps fees to the feeTo address if there are more than 0 wei for
+     * each address in tokens.
+     * @param tokens The tokens for which fees will be swept to the feeTo address.
+     */
     function sweepFees(address[] memory tokens) external;
 
-    // @notice Create a new options chain from optionInfo if it doesn't already exist
+    /**
+     * @notice Create a new options chain from optionInfo if it doesn't already exist
+     * @param optionInfo The optionInfo from which a new chain will be created, with
+     * settlementSeed set to 0.
```

Comment:
Is it set to 0, or is it set to the hash of the option?

Reply:
The resultant chainKey is the output of the kecak256 hash of the option with the seed set to zero. But internally, it's set to the hash of the option. I can clarify this in the comment.

Reply:
@0xAlcibiades updated, lmk if that new comment clarifies it sufficiently.